apps/projects/app/script.js

```diff
@@ -4,17 +4,12 @@ import { first } from 'rxjs/operators'
 import { retryEvery } from '../../../shared/ui/utils'
 import { app, initStore } from './store'
 
-retryEvery(async retry => {
+retryEvery(async () => {
```

Comment:
If we're removing retry here I have a feeling using the retryEvery function here doesn't even make sense/we're not really using it.

Reply:
Yeah, that seemed weird to me! Should I get rid of `retryEvery` while here? What should I replace it with?

Reply:
I think a good approach to still have the `retryEvery` feature is to pass it as param to the child function, as we did in Allocations, I am not that sure about the real utility of this helper function, but every core aragon app uses it.

I would say keep it like Allocations app and check if some warnings about `localhost:4444 postMessage origin = null` get solved